docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
```

Comment:
Should tweak it smth like
```suggestion

export LC_ALL=C
set -e
```
to make linter happy

Reply:
OK, added. What linter are you using? I'm not seeing this warning in mine.

Reply:
https://github.com/dashpay/dash/blob/master/test/lint/lint-shell-locale.sh